tests/sandbox/test_sandbox.py

```diff
@@ -1,57 +1,57 @@
 import pytest
 from aiohttp import ClientResponse
 from .data.scenarios import relatedPerson, retrieve, search, update
+import requests
+from typing import Dict
 from .utils import helpers
-from api_test_utils import env
-from api_test_utils import poll_until
-from api_test_utils.api_session_client import APISessionClient
-from api_test_utils.api_test_session_config import APITestSessionConfig
 
 
 @pytest.mark.deployment_scenarios
 class TestPDSSandboxDeploymentSuite:
     """Sandbox PDS Deployment Scenarios. Checks performed: status_codes and version deployed"""
 
     @pytest.mark.asyncio
-    async def test_wait_for_ping(self, api_client: APISessionClient, api_test_config: APITestSessionConfig):
+    async def test_wait_for_ping(self,
+                                 commit_id: str,
+                                 nhsd_apim_proxy_url: str):
```

Comment:
Given time contraints feel free to ignore, but might be worth adding some doc strings explaining the test case like we have in the spine repo

Reply:
I followed the commenting style of the rest of the file and other test_* files: don't comment. Hopefully the tests and their calls can be understood without them. I think I prefer this style. Makes me more cautious with how I am naming things.